source/reference/kubectl-minio-plugin/kubectl-minio-init.rst

```diff
@@ -0,0 +1,150 @@
+
+.. _kubectl-minio-init:
+
+=========================
+``kubectl minio init``
+=========================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: kubectl minio init
+
+Description
+-----------
+
+.. start-kubectl-minio-init-desc
+
+The :mc:`kubectl minio init` command initializes the MinIO Operator. 
+:mc:`kubectl minio init` requires the operator for core functionality.
+
+.. end-kubectl-minio-init-desc
+
+Syntax
+------
+
+.. tab-set::
+
+   .. tab-item:: EXAMPLE
+
+      The following command initializes a new MinIO Operator deployment using operator v4.4.22, on the ``minio-operator`` namespace, using the cluster domain of ``cluster.local`` and watching the default namespace.:
+
+      .. code-block:: shell
+         :class: copyable
+
+         kubectl minio init                      \
+             --image=minio/operator:v4.4.22      \
+             --namespace=minio-operator          \
+             --cluster-domain=cluster.local      \
+             --namespace-to-watch=default        \
+             --image-pull-secret=  
+
+   .. tab-item:: SYNTAX
+
+      The command has the following syntax:
+
+      .. code-block:: shell
+         :class: copyable
+
+         kubectl minio init                      \
+                       [--cluster-domain]        \
+                       [--console-image]         \
+                       [--default-console-image] \
+                       [--default-kes-image]     \
+                       [--default-minio-image]   \
+                       [--image]                 \
+                       [--image-pull-secret]     \
+                       [--namespace]             \
+                       [--namespace-to-watch]    \
+                       [--output]                \
+                       [--prometheus-name]       \
```

Comment:
the prometheus flags are not printed in  help (i.e `kubectl minio init -h`)

Reply:
They are for me on 4.4.23. What version are you looking at @prakashsvmx ?